update_prevalence.py

```diff
@@ -894,7 +937,7 @@ def main() -> None:
         data.populate_fips_cache()
 
         # Cumulative cases per region
-        populate_since = effective_date - timedelta(days=15)
+        populate_since = effective_date - timedelta(days=16)
```

Comment:
Should this still be 15? The code above only looks at index -15 which is 14 elements.

Reply:
-15 is 15 elements. You need the 15th day to calculate 7 days worth of cases from cumulative data.

I think it is indeed safe to change this to -15, though.

Reply:
on Sunday there were 10 cases (cumulative).
on Monday there were 12.
on Tuesday there were 13.
on Wednesday there were 17.

to calculate the number of new cases in 3 days, we need to subtract the Sunday count (index -4) from the Wednesday count (index -1). Otherwise you miss the 2 cases that happened between the end of Sunday and the end of Monday.